sw/device/tests/keymgr_sideload_aes_test.c

```diff
@@ -92,7 +127,7 @@ void aes_test(void) {
   // Trigger the AES decryption and wait for it to complete.
   CHECK_DIF_OK(dif_aes_trigger(&aes, kDifAesTriggerStart));
   AES_TESTUTILS_WAIT_FOR_STATUS(&aes, kDifAesStatusOutputValid, true, TIMEOUT);
-
+  AES_TESTUTILS_WAIT_FOR_STATUS
```

Comment:
This line does not seem right, it shouldn't even compile.

Reply:
I know :)! It was a glitch at my end locally, and I was fixing and updating.